discord-scripts/fix-twitter-embed.ts

```diff
@@ -62,27 +67,37 @@ async function workingTwitterEmbeds(
 //
 // See https://github.com/FixTweet/FxTwitter for more.
 export default function fixTwitterEmbeds(discordClient: Client, robot: Robot) {
+  // Process only messages that match the Twitter URL pattern
   discordClient.on("messageCreate", (message) => {
-    robot.logger.info(
-      `fixTwitterEmbeds: processing new message ${message.content}`,
-    )
-
-    workingTwitterEmbeds(message, robot.logger).catch((err) => {
-      robot.logger.error(
-        `fixTwitterEmbeds: failed to process new message ${message.content}: ${err}`,
+    if (message.content?.match(twitterUrlRegExp)) {
+      robot.logger.info(
+        `fixTwitterEmbeds: processing new message ${message.content}`,
```

Comment:
We should also probably never log message content unless we're debugging.

Instead, let's log the channel, `from` user, and timestamp.

Reply:
Also we should maybe still always log that we're processing every message, but at the `debug` level, and making sure that production isn't set to show debug level logs...

Reply:
Sounds good and should I also look through other helpers and move any `logger.info` over to `logger.debug` when we want to separate that? On a case-by-case basis ofc!

Reply:
Yes, def makes sense to adjust log level.

What we need to peek at is what changes we might need to make so that debug shows on local testing instances but not on production instances.

Reply:
I ended up adding `robot.logger.debug` to the mix so that we only log those if this flag is set `export HUBOT_LOG_LEVEL="debug"` which is nicely already built into Hubot! This is probably a good time to start moving other scripts over to the debug level as well. 🥲

Reply:
We still need to not log message content on info level though. I think we want to:
- Change this log to only log channel, `from`, and timestamp.
- *Maybe*, add a debug log that logs the message content.